stretch_core/launch/stretch_robot.launch.py

```diff
@@ -0,0 +1,41 @@
+from ament_index_python.packages import get_package_share_directory
+
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument, IncludeLaunchDescription
+from launch.conditions import IfCondition, UnlessCondition
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import LaunchConfiguration
+
+
+def generate_launch_description():
+    ld = LaunchDescription()
+    package_prefix = get_package_share_directory('stretch_core')
+
+    driver_py = PythonLaunchDescriptionSource([package_prefix, '/launch/stretch_driver.launch.py'])
+    ld.add_action(IncludeLaunchDescription(driver_py, launch_arguments={'broadcast_odom_tf': 'true'}.items()))
+
+    resolution_arg = DeclareLaunchArgument(
+        'high_res',
+        default_value='False',
+        description='Whether to use high resolution realsense'
+    )
+
+    hi_realsense_py = PythonLaunchDescriptionSource([package_prefix, '/launch/d435i_high_resolution.launch.py'])
+    hi_realsense_launch = IncludeLaunchDescription(hi_realsense_py,
+                                                   condition=IfCondition(LaunchConfiguration('high_res')))
+
+    lo_realsense_py = PythonLaunchDescriptionSource([package_prefix, '/launch/d435i_low_resolution.launch.py'])
+    lo_realsense_launch = IncludeLaunchDescription(lo_realsense_py,
+                                                   condition=UnlessCondition(LaunchConfiguration('high_res')))
+    ld.add_action(resolution_arg)
+    ld.add_action(hi_realsense_launch)
+    ld.add_action(lo_realsense_launch)
+
+    rplidar_py = PythonLaunchDescriptionSource([package_prefix, '/launch/rplidar.launch.py'])
+    rplidar_launch = IncludeLaunchDescription(rplidar_py)
+    ld.add_action(rplidar_launch)
+
+    # TODO: Include respeaker
+    # TODO: Include ekf
```

Comment:
Do you want me to open issues for these two?

Reply:
Eh, not needed at the moment, I think.